detectable/src/main/java/com/synopsys/integration/detectable/detectables/conda/parser/CondaListParser.java

```diff
@@ -24,34 +24,44 @@
 import com.synopsys.integration.detectable.detectables.conda.model.CondaListElement;
 
 public class CondaListParser {
+    private static final String PYPI_CHANNEL = "pypi";
```

Comment:
There are.  In the research I did, though, I couldn't find mention/use of any other common channel that would require a different forge than ANACONDA.  I figure if such channels exist/we are dropping matches because customers are pulling components from them, we'll handle those ad hoc.

Reply:
With that knowledge I would try to write this such that it's ready to accept more channels. Perhaps a map of channel names to forges.

Reply:
I think it could be tough to create a data structure that accomplishes what we want without an if/else block or essentially reproducing condaListElementToDependency.  The reason being that an element's channel doesn't just have implications for an external ID's forge, but also its version.  I think its reasonable to extract this method to another class, though